app/cdap/components/Administration/TetheringTabContent/CdfTetheringConnections/index.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import T from 'i18n-react';
+import NewRequests from './NewRequests';
+import Connections from './Connections';
+
+const PREFIX = 'features.Administration';
+const I18NPREFIX = `${PREFIX}.Tethering`;
+
+const CdfTetheringContainer = styled.div``;
```

Comment:
why have all these non-styled divs? Are you doing this to add context or something?